src/System.Diagnostics.Tools/tests/System/Diagnostics/CodeAnalysis/SuppressMessageAttributeTests.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace System.Diagnostics.CodeAnalysis.Tests
+{
+    public static class SuppressMessageAttributeTests
+    {
+        [Fact]
+        public static void TestConstructorBasic()
+        {
+            string expectedCategory = "Category";
+            string expectedCheckId = "CheckId";
+            SuppressMessageAttribute sma = new SuppressMessageAttribute(expectedCategory, expectedCheckId);
+            Assert.Equal(expectedCategory, sma.Category);
+            Assert.Equal(expectedCheckId, sma.CheckId);
+        }
+
+        [Fact]
+        public static void TestConstructorExtended()
```

Comment:
This isn't really testing the constructor any more than TestConstructorBasic is.  This is really a test of the additional properties on the attribute to make sure that they can be set and then have the same value retrieved.